pom.xml

```diff
@@ -286,7 +286,7 @@
             <dependency>
                 <groupId>ch.qos.logback</groupId>
                 <artifactId>logback-classic</artifactId>
-                <version>1.4.7</version>
+                <version>1.3.4</version>
```

Comment:
Why the version downgrade?

Reply:
because of this: https://github.com/awslabs/aws-lambda-powertools-java/actions/runs/5165759332/jobs/9305470825#step:5:382

Reply:
Got it. Seems that 1.3.X should support Java 8. Any issues with the 1.3.7?
https://mvnrepository.com/artifact/ch.qos.logback/logback-classic/1.3.7

Reply:
right... for some reason WebStorm proposed the 1.3.4... Just updated with 1.3.7